### ISSUE TITLE
Events inline

### ISSUE THREAD

**Issue by thiagoducca** [fullcalendar/fullcalendar-scheduler#422]
I'm creating a Rental Property Management solution and I would like to use the Scheduler with the month view, where each event has a starting timestamp and an ending timestamp. For example, imagine a  reservation that starts on "2018-01-01 at 4pm" and finishes on "2018-01-10 at 11am".

In the same day, imagine we have another reservation that starts on "2018-01-10 at 4pm" and finishes on "2018-01-20 at 11pm".

I would like to view both those events in the same line, same horizontal alignment, sharing the same day slot, without dropping to the next line or any other kind of indentation. Is that possible?

I have been researching others plugins and I found this one as an example of what I need: https://dhtmlx.com/docs/products/demoApps/car-rental-html5-js-php/

Visual example of what I mean: ![untitled-2](https://user-images.githubusercontent.com/8313259/36238467-b2094196-11d0-11e8-8c25-c92c354d1ef0.jpg)

In conclusion, I would like for items to follow the same straight "day line" when they happen in the same day. 

Thank you!

**Comment by thiagoducca** [comment]
hey there @acerix, thank you for your reply! to clarify my issue, check this codepen: https://codepen.io/anon/pen/VQrLVQ?editors=0010 - basically, it seems that `{eventOverlap: false}` doesn't work for the `timelineMonth` view, as I would expect both reservations on "2323 South Lane" to show up one after the other (where they are shown one above the other). How can I make them show up one right after the other, assuming overlapping dates are guaranteed not to happen?"

![untitled-4](https://user-images.githubusercontent.com/8313259/36273294-0ce53800-1252-11e8-99e9-afc3f491a99f.jpg)

https://user-images.githubusercontent.com/8313259/36238467-b2094196-11d0-11e8-8c25-c92c354d1ef0.jpg

**Comment by MartijnWelker** [comment]
@thiagoducca when the [`slotDuration`](https://fullcalendar.io/docs/agenda/slotDuration/) property is set to 24 hours it will render events as full day events, by lowering the `slotDuration` you can achieve what you want.

Updated pen here
https://codepen.io/anon/pen/OQvgwB?editors=0010
